lib/Dialect/Ibis/IbisOps.cpp

```diff
@@ -141,6 +141,143 @@ LogicalResult ReturnOp::verify() {
 
   return success();
 }
+
+//===----------------------------------------------------------------------===//
+// InstanceOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult InstanceOp::verifySymbolUses(SymbolTableCollection &symbolTable) {
+  auto targetClass = getClass(&symbolTable);
+  if (!targetClass)
+    return emitOpError() << "'" << getClassName() << "' does not exist";
+
+  return success();
+}
+
+//===----------------------------------------------------------------------===//
+// PortReadOp
+//===----------------------------------------------------------------------===//
+
+template <typename TSrcOp, typename TTargetOp>
+LogicalResult verifyPortSymbolUses(
+    TSrcOp op, StringAttr symName, llvm::function_ref<Type(TSrcOp)> getPortType,
+    SymbolTableCollection &symbolTable,
```

Comment:
Do you use symbolTable anywhere?

Reply:
I'd imagine you planned on passing it into getTargetOp but ended up using captures instead.